<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainDialog</class>
 <widget class="QDialog" name="MainDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>873</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>客户端与服务器架构系统-TCP客户端 Version 1.8</string>
  </property>
  <property name="windowIcon">
   <iconset resource="images.qrc">
    <normaloff>:/new/prefix1/images/logo.ico</normaloff>:/new/prefix1/images/logo.ico</iconset>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>服务器IP地址：</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>20</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>服务器端口：</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox_ServerIP">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>20</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </item>
  </widget>
  <widget class="QSpinBox" name="spinBox_ServerPort">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>20</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="maximum">
    <number>65535</number>
   </property>
   <property name="value">
    <number>6688</number>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_ConnectServer">
   <property name="geometry">
    <rect>
     <x>590</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>连接服务器</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/connect.ico</normaloff>:/new/prefix1/images/connect.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_DisconnectServer">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>20</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>断开服务器</string>
   </property>
   <property name="icon">
    <iconset resource="images.qrc">
     <normaloff>:/new/prefix1/images/disconnect.ico</normaloff>:/new/prefix1/images/disconnect.ico</iconset>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_SendMsg">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>831</width>
     <height>121</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>851</width>
     <height>51</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>851</width>
     <height>221</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>消息列表</string>
   </property>
   <widget class="QPlainTextEdit" name="plainTextEdit_MsgList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>831</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>300</y>
     <width>851</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>发送消息</string>
   </property>
   <widget class="QPushButton" name="pushButton_Exit">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>退出系统</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/prefix1/images/exit.ico</normaloff>:/new/prefix1/images/exit.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_ClearMsg">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>清除消息</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/prefix1/images/clear.ico</normaloff>:/new/prefix1/images/clear.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SendMsg">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>150</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发送消息</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/prefix1/images/sendmsg.ico</normaloff>:/new/prefix1/images/sendmsg.ico</iconset>
    </property>
   </widget>
   <widget class="QLabel" name="label_LocalIP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>本机IP地址</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>500</y>
     <width>851</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>仿宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="title">
    <string>传输文件至服务器</string>
   </property>
   <widget class="QPushButton" name="pushButton_SendFile">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>发送文件</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/prefix1/images/sendfile.ico</normaloff>:/new/prefix1/images/sendfile.ico</iconset>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_SelectFile">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>20</y>
      <width>121</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>选择文件</string>
    </property>
    <property name="icon">
     <iconset resource="images.qrc">
      <normaloff>:/new/prefix1/images/openfile.ico</normaloff>:/new/prefix1/images/openfile.ico</iconset>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>831</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>仿宋</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
  </widget>
  <zorder>groupBox_4</zorder>
  <zorder>groupBox_3</zorder>
  <zorder>groupBox_2</zorder>
  <zorder>groupBox</zorder>
  <zorder>label</zorder>
  <zorder>label_2</zorder>
  <zorder>comboBox_ServerIP</zorder>
  <zorder>spinBox_ServerPort</zorder>
  <zorder>pushButton_ConnectServer</zorder>
  <zorder>pushButton_DisconnectServer</zorder>
  <zorder>plainTextEdit_SendMsg</zorder>
 </widget>
 <resources>
  <include location="images.qrc"/>
 </resources>
 <connections/>
</ui>
